db/migrate/20250603164430_update_tenant_name.rb

```diff
@@ -0,0 +1,23 @@
+class UpdateTenantName < ActiveRecord::Migration[7.0]
+  class SettingsChange < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+  end
+
+  class Tenant < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+  end
+
+  def change
+    say_with_time("Updating root tenant's name from settings") do
+      # there may be multiple of these, but it should only be
+      sc = SettingsChange.in_my_region.find_by(:key => "/server/company")
+      Tenant.find_by(:ancestry => nil, :use_config_for_attributes => true)&.update(:name => sc&.value || "My Company", :use_config_for_attributes => false)
```

Comment:
I think there's a bug here? What about the case where the Tenant has a name _and_ there isn't a value in /server/company. Then I think this changes the Tenant name to whatever the user set to "My Company".  Is that even an expected combination?